src/main/java/Domain/Game/Team.java

```diff
@@ -98,16 +100,19 @@ public boolean addTeamManager(TeamOwner townr,Role teamManager)
         return false;
     }
 
-    public boolean addTeamOwner(TeamOwner townr,TeamOwner teamOwner)
+    public boolean addTeamOwner(TeamOwner teamOwner)
```

Comment:
Why did you changed it? the function should receive 2 TeamOwners, one who is already part of the Team Owners and the 2nd is the new Owner that should be added

Reply:
השאלה היא האם יכול לקרות מצב שיוצרים קבוצה ללא בעלים ואז בפעם הראשונה שתרצה להוסיף בעל קבוצה מה יקרה? לא יהיה לך בעל קבוצה לשים. הTeamController בודק את הבדיקה של האם זה שביקש את ההוספה של בעל קבוצה חדש הוא בעל קבוצה בעצמו.

Reply:
אהה סבבה, אז הבדיקה בעצם מתבצעת בקונטרולר במקום בקבוצה ....